axum/src/extract/rejection.rs

```diff
@@ -65,6 +65,14 @@ define_rejection! {
     pub struct InvalidFormContentType;
 }
 
+define_rejection! {
+    #[status = BAD_REQUEST]
```

Comment:
I didn't really know which status to use, this probably isn't ideal

Reply:
I think `Bad Request` is appropriate.